spec/std/time/time_spec.cr

```diff
@@ -60,6 +60,16 @@ describe Time do
     time.epoch_ms.should eq(milliseconds)
   end
 
+  it "returns always increasing monotonic clock" do
+    clock = Time.monotonic
+    Time.monotonic.should be >= clock
+  end
+
+  it "measures elapsed time using monotonic clock" do
+    span = Time.measure { sleep 1.millisecond }
+    span.should be >= 1.millisecond
```

Comment:
I would put an upper bound of 2-5ms on this.

Reply:
I'm not sure to understand. Can you detail?

Reply:
Just chack that span is `1.millisecond <= span <= 10.milliseconds`. If we get a weird value like `1.second` after a `sleep 1.millisecond` we should probably fail.

Reply:
Oh no no no. The test is currently in a much better state than this suggestion of a flaky test. Again you overestimate the accuracy of system timers.
At least on Windows I know that sleeping for 1 millisecond can skip the sleep entirely [1], and that the resolution at least on some old system that I used was about 12 ms.

\[1]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms686298(v=vs.85).aspx
> If dwMilliseconds is less than the resolution of the system clock, the thread may sleep for less than the specified length of time

Reply:
Ok, use 100ms then. If your 1ms sleep turns into a 100ms sleep then your system is broken and its pointless running specs on it regardless.

Reply:
This isn't production code, this is the spec suite. If a test fails nothing breaks, unless we happen to be using such broken machines on CI.

Reply:
Looking at the CI failures, I guess we can go for something like `sleep 30; 20 <= span <= 100`